common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -167,6 +231,11 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     Utils.timeStringAsSeconds(get(key, defaultValue))
   }
 
+  /** Get time in seconds for a transport config */
+  def getTransportConfTimeAsSeconds(module: String, configEntry: ConfigEntry[Long]): Long = {
```

Comment:
This method is not used.

Reply:
Agree, I added those for completeness sake - but I can remove all those which are not used.
Will update the PR

Reply:
Done